webconsole/install.go

```diff
@@ -1,26 +0,0 @@
-package webconsole
-
-import (
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-
-	webconsolev1 "github.com/openshift/api/webconsole/v1"
-)
-
-const (
-	GroupName = "webconsole.openshift.io"
```

Comment:
We can't remove stuff just like that b/c we don't use it. We need to deprecate it first and follow the usual guidelines. Upstream has 3 releases or 1 year whichever is longer, not sure what about us? /cc @deads2k

Reply:
Thanks, that's definitely why I opened this.  We don't use it, so it should at least be deprecated and planned to remove in the future if possible.

Reply:
Resolving per @deads2k comment confirming that we can remove.